src/store/reducers/bookingsReducer.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+
+const GET_BOOKINGS = 'GET_BOOKINGS';
+
+const POST_BOOKING = 'POST_BOOKING';
+
+const getBookings = bookings => {
+    return { type: GET_BOOKINGS, bookings }
+}
+
+const addBooking = booking => {
+    return { type: POST_BOOKING, booking }
+}
+
+export const fetchBookings = userId => {
+    return dispatch => {
+        axios.get(`/api/users/${userId}/bookings`)
+            .then(res => res.data)
+            .then(bookings => {
```

Comment:
nit: you can combine these two `.then`'s since the first is only accessing `.data`